css/form.css

```diff
@@ -1524,6 +1524,10 @@ form hr {
     display: flex !important;
     align-items: center;
   }
+
+  .scanner.btn.btn-primary {
+    margin-bottom: 8px;
```

Comment:
Why is this here? Doesn't seem related to custom field widths.

Reply:
Removed!